.vscode/launch.json

```diff
@@ -15,6 +15,40 @@
       "preLaunchTask": "Debug: Desktop",
       "postDebugTask": "Stop Debug",
       "outFiles": [ "${workspaceFolder}/dist/**/*.js" ]
-    }
+    },
+    { 
+      "type": "node", 
+      "request": "launch", 
+      "name": "Mocha Tests", 
+      "program": "${workspaceFolder}/node_modules/mocha/bin/_mocha", 
+      "args": [ 
```

Comment:
Use VS Code `Format Document` to clean up indentation.

Is Attach To Process needed here?